com.w1hkj.fldigi.yml

```diff
@@ -11,7 +11,6 @@ finish-args:
   - --socket=pulseaudio
   - --persist=.fldigi
   - --persist=.fltk
-  - --persist=.nbems
```

Comment:
Why did you delete the `.nbems` persistent directory? It was not used by flcluster, however it is used by fldigi.

Reply:
Consider reverting this change.

Reply:
@lu9dce **^** One more thing to fix. ;-)

Reply:
- --persist = .nbems
I already took it out
add it again?

Reply:
Yes.